### DISCUSSION TITLE
Standard Metadata parser / format between O.S

### DISCUSSION THREAD

**Discussion by Arkamis** [hasura/graphql-engine#9192] [Question]
# Description

Hello, I'm using hasura-cli with a github hosted repo. We are currently 2 developers on my team. He uses a windows machine with a subsystem for Linux.  

Every metadata change using your web interface generated by Hasura console changes the format of Hasura metadata.

# Examples:
### git changes before and after adding a new field to an existing table.
![image](https://user-images.githubusercontent.com/37283793/200643522-c1dc3b96-25a7-4f71-a6d9-becb951e468a.png)
After adding a new column to one of my tables.
### git file YAMLs files change some field positions.
![image](https://user-images.githubusercontent.com/37283793/200643869-28e6cdc1-bbb1-438d-86a1-b20d591bf6a3.png)

Every other existing file has similar changes...

Does anybody have any thoughts on this?

**Comment by adas98012** [comment]
@Arkamis 
This is expected if you are running Hasura GraphQL engine locally (using Docker). You can check-in the changes so that the metadata changes would be persistent when the github branch is deployed elsewhere.
